### PR TITLE
Sus 180/adding access endpoints

### PR DESCRIPTION
**Pull request by JiaanNie**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: src/services/sam/access/access.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  getKey(keyUuid: string): Promise<GetKeyResponse> {
+    return this.axiosClient.post<GetKeyResponse>(
+      `/sam/security/keys/${keyUuid}`
+    ).then((res: AxiosResponse<GetKeyResponse>) => {
+      this.logger.debug(res);
+      return res.data;
+    });
+  }
+
+  checkAccessibility(partition: string, target: string, node: string, entity: string): Promise<GetAccessibilityResponse> {
```
Comment (ShawnClake) [inline src/services/sam/access/access.ts]: Can we rename this to `checkAccess`, same with it's models (`GetAccessResponse`)

**Review by ShawnClake — CHANGES_REQUESTED** [review]
There's a couple empty files in this PR. Let's remove those files and then we can re-add them when we need them down the line

File: src/services/sam/access/types/accessResponseTypes.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export interface GetKeyResponse {
+  uuid: string,
+  access: string,
+  name: string,
+  summary: string,
+  created_at: string,
+  last_used_at: string
+}
+
+export interface GetAccessibilityResponse{
```
Comment (ShawnClake) [inline src/services/sam/access/types/accessResponseTypes.ts]: You may want to verify that the format of this response is correct. I could've sworn this endpoint only returns a bool

Comment (JiaanNie) [reply src/services/sam/access/types/accessResponseTypes.ts]: it return a http_200_ok object which pass in a bool in `responses.http_200_ok(True)`

Comment (JiaanNie) [reply src/services/sam/access/types/accessResponseTypes.ts]: should i still return a bool then?

**Review by ghost — APPROVED** [review]

**Pull request merged**